.github/workflows/retryable-monitor.yml

```diff
@@ -0,0 +1,17 @@
+name: Retryable Monitor
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"  # Run every 24 hours at midnight UTC
```

Comment:
The original monitor runs at `- cron: "3 8 * * *"` along with ArbitrumOne retryable monitor. We should maintain that.